test/plugin-stripe/config.ts

```diff
@@ -1,4 +1,4 @@
-import stripePlugin from '@payloadcms/plugin-stripe'
+import { stripePlugin } from '@payloadcms/plugin-stripe'
```

Comment:
Can we still export as default from the package to avoid the breaking change? You can keep the export how you did it in the /src/admin.ts file and imported just to export from the package as default.